flow_transferFiles2Apollo/createOrganism-TF.cwl

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+baseCommand: [createOrganism.py]
+
+$namespaces:
+  cwltool: "http://commonwl.org/cwltool#"
+
+hints:
+  cwltool:LoadListingRequirement:
+    loadListing: shallow_listing
+
+arguments: 
+  - position: 1
+    prefix: -host
+    valueFrom: $(inputs.host[0])
+  - position: 3
+    prefix: -organism
+    valueFrom: $(inputs.scientific_name[0]) $(inputs.scientific_name[1])
+  - position: 5
+    prefix: -genus
+    valueFrom: $(inputs.scientific_name[0])
+  - position: 7
+    prefix: -species
+    valueFrom: $(inputs.scientific_name[1])
+  - position: 9
+    prefix: -directory
+    valueFrom: $(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_apollo2_data[0])
+  - position: 11
+    prefix: -blatdb
+    valueFrom: $(inputs.PATH[2])/$(inputs.tree[0])/*.2bi
```

Comment:
The .2bi file needs to be specifically referenced here (instead of a wildcard) - there can be > 1 .2bi file in a directory, and the blat function in Apollo won't work if you get the wrong one. You can probably use the scaffold base name to make the .2bi file name.

Reply:
Can I change it to "$(inputs.PATH[2])/$(inputs.tree[0])/*$(inputs.tree[1])_genomic.fna.2bi"?
Is there any file that may have "Apis_HAv3.1_genomic.fna.2bi" in its name?